modules/specifying-oc-log-levels.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/diagnosing-oc-issues.adoc
+
+[id="specifying-oc-log-levels_{context}"]
+= Specifying OpenShift CLI (`oc`) log levels
+
+You can investigate OpenShift CLI (`oc`) issues by increasing the command's log level.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
```

Comment:
No need to as a user with cluster-admin role, normal user also could use loglevel.

Reply:
Thank you @zhouying7780 for highlighting this. I have now resolved this by removing that prerequisite.